src/controller/storagemgr/storage.go

```diff
@@ -0,0 +1,46 @@
+package storagemgr
+
+import (
+	"controller/storagemgr/storagedriver"
+	"errors"
+	"github.com/edgexfoundry/device-sdk-go"
+	"github.com/edgexfoundry/device-sdk-go/pkg/startup"
+	"log"
+	"os"
+)
+
+
```

Comment:
Any reason to put multiple blank lines here?

Reply:
No, I will remove them.